cmake/options.cmake

```diff
@@ -10,7 +10,7 @@ endif ()
 option(SCN_TESTS "Enable tests" ${SCN_ENABLE_EXTRAS})
 option(SCN_DOCS "Enable docs target" ${SCN_ENABLE_EXTRAS})
 option(SCN_EXAMPLES "Enable examples target" ${SCN_ENABLE_EXTRAS})
-option(SCN_INSTALL "Enable install target" ${SCN_ENABLE_EXTRAS})
+option(SCN_INSTALL "Enable install target" ON)
```

Comment:
I think it makes sense for a super project to install a subproject by default. If this breaks existing uses cases this commit can be dropped and the beman superproject can set this option.

Reply:
On second thought, I think this is a bad idea and I have reverted this change. I expect the typical usage is to link statically and installing the static library would be undesirable.